README.md

```diff
@@ -56,6 +56,15 @@ This will output A LOT of additional logs to the console. This also disable mini
 
 You can read more about the configuration in the [docs](https://opennext.js.org/aws/config)
 
+## Preleases
+
+Besides the standard npm releases we also automatically publish prerelease packages on branch pushes (using [`pkg.pr.new`](https://github.com/stackblitz-labs/pkg.pr.new)):
+
+- `https://pkg.pr.new/@opennextjs/aws@main`:
+  Updated with every push to the `main` branch, this prerelease contains the most up to date yet (reasonably) stable version of the package.
+- `https://pkg.pr.new/@opennextjs/aws@experimental`
+  Updated with every push to the `experimental` branch (when present), this prerelease contains the latest experimental version of the package (containing features that we want to test/experiment on before committing to).
+  Which you can simply install directly with your package manager of choice, for example: `npm i https://pkg.pr.new/@opennextjs/aws@main`
```

Comment:
```suggestion
  Which you can simply install directly with your package manager of choice, for example: `npm i https://pkg.pr.new/@opennextjs/aws@experimental`
```

Reply:
Sorry I accepted the suggestion but then realised that this is probably not really what we want, since the `experimental` prerelease might not be the always relevant (like when there is no "experimentation" going on etc... and the `experimental` branch either doesn't exist or is out of date).

using `main` here does seem the right choice to me, I also noticed that I messed up the formatting 🤦

This is how it looks now:
![Screenshot 2024-11-01 at 09 50 57](https://github.com/user-attachments/assets/4e475287-fb59-4316-978c-f4676e77a80f)

Hopefully this is fine? 🙂